Update  Replacer parameter definition (#23093)
mdn/content: files/en-us/web/javascript/reference/global_objects/json/stringify/index.md

- `value`
  - : The value to convert to a JSON string.
- `replacer` {{optional_inline}}

>   - : A function that alters the behavior of the stringification process, or an array of strings and numbers that acts as a approved list for selecting the object properties that will be included in the output. If `replacer` is an array, all elements that are not strings or numbers (can be either primitives or wrapper objects), including {{jsxref("Symbol")}} values, are completely ignored. If `replacer` is anything other than a function or an array (e.g. [`null`](/en-US/docs/Web/JavaScript/Reference/Operators/null) or not provided), all properties of the object are included in the resulting JSON string.

Reviewer: ```suggestion
  - : A function that alters the behavior of the stringification process, or an array of strings and numbers that specifies properties of `value` to be included in the output. If `replacer` is an array, all elements that are not strings or numbers (can be either primitives or wrapper objects), including {{jsxref("Symbol")}} values, are completely ignored. If `replacer` is anything other than a function or an array (e.g. [`null`](/en-US/docs/Web/JavaScript/Reference/Operators/null) or not provided), all string-keyed properties of the object are included in the resulting JSON string.
```
